src/main/java/dev/dokan/dokan_java/conv/MaskValueEnum.java

```diff
@@ -0,0 +1,21 @@
+package dev.dokan.dokan_java.conv;
+
+
+public interface MaskValueEnum extends EnumInteger {
+
+    static <T extends Enum<T> & MaskValueEnum> T enumFromInt(final int value, final T[] enumValues) {
+        return EnumInteger.enumFromInt(value, enumValues);
+    }
+
+    static <T extends Enum<T> & MaskValueEnum> MaskValueSet<T> getSetFromInt(final int intValue, final T[] allEnumValues) {
+        return MaskValueSet.getSetFromInt(intValue, allEnumValues);
+    }
+
+    static <T extends Enum<T> & MaskValueEnum> MaskValueSet<T> getSetFromInt(final int intValue, final Class<T> type) {
```

Comment:
I'm asking myself currently, if these methods should or shouldn't be part of the MasValueEnum class.

They couple the MaskValueSet, which relies on this interface, to the interface itself and reverse the dependency.
If it is implemented in the base class, it is okay, because the type is directly fixed and inreases type safety, but the base interface should not contain any reference to MaskValueSet

Reply:
I was thinking about that a lot, too.
I decided to throw it in because `MaskValueEnum` and `MaskValueSet` have cyclic dependency on each other. MaskValueSet is designed to specifically only hold MaskValueEnums and MaskValueEnum is designed to be used with MaskValueSet. 

Also putting the `maskValueSet` method into MaskValueEnum creates a contract between it and all implementing classes showing where the same method "comes from" in, e.g. MountOption. 
At least that was my line of thinking here. Java sadly doesn't support static inheritance.

Reply:
I vote against it.
An alternative can be having an inner interface like in the case of [Map and Map.Entry<K,V>](https://www.programcreek.com/2013/08/inner-interface-in-java/).

Reply:
@infeo and I agreed to remove them entirely.